pyanaconda/core/constants.py

```diff
@@ -176,6 +176,25 @@
     "No disks selected; please select at least one disk to install to."
 )
 
+# Kernel messages.
+WARNING_SMT_ENABLED_GUI = N_(
+    "Simultaneous Multithreading (SMT) technology can provide performance "
+    "improvements for certain workloads, but introduces several publicly "
+    "disclosed security issues. You have the option of disabling SMT, which "
+    "may impact performance. If you choose to leave SMT enabled, please read "
+    "https://red.ht/rhel-smt to understand your potential risks and learn "
+    "about other ways to mitigate these risks."
+)
+
+# This message is shorter to fit on the screen.
+WARNING_SMT_ENABLED_TUI = N_(
+    "Simultaneous Multithreading (SMT) may improve performance for certain "
+    "workloads, but introduces several publicly disclosed security issues. "
+    "You can disable SMT, which may impact performance. Please read "
+    "https://red.ht/rhel-smt to understand potential risks and learn about "
```

Comment:
If we want to show this in Fedora pointing to access.redhat.com (which is where this shortened URL goes) might look a bit strange, but on the other hand I have not been able to find anything neutral that would be as authoritative.

Reply:
I'm not sure if we want to show this in Fedora at all. Isn't this just a RHEL decision?

Reply:
Yeah, that in the end seems like a simpler solution.